internal/plugin/config_plugin.go

```diff
@@ -67,7 +71,15 @@ func (c *Config) Process(ctx context.Context, msg *bus.Message) {
 		c.processInstanceConfigUpdateRequest(ctx, msg)
 	case msg.Topic == bus.InstancesTopic:
```

Comment:
No need to listen to instances topic since the resource plugin is adding the instances to the resource

Reply:
was gonna keep them separate since the instances might change more than the resource does